src/sentry/api/endpoints/internal_beacon.py

```diff
@@ -0,0 +1,60 @@
+import logging
+
+from django.conf import settings
+from rest_framework.response import Response
+
+from sentry import get_version, is_docker, options
+from sentry.api.base import Endpoint
+from sentry.http import safe_urlopen
+from sentry.tasks.beacon import BEACON_URL
+
+logger = logging.getLogger("beacon")
+
+
+class InternalBeaconEndpoint(Endpoint):
+    permission_classes = ()
+
+    def post(self, request):
+        error = None
+        install_id = options.get("sentry:install-id")
+
+        if not settings.SENTRY_BEACON:
+            logger.info(
+                "beacon_metric.skipped", extra={"install_id": install_id, "reason": "disabled"}
+            )
+            return Response(status=204)
+
+        if settings.DEBUG:
+            logger.info(
+                "beacon_metric.skipped", extra={"install_id": install_id, "reason": "debug"}
+            )
+            return Response(status=204)
+
+        anonymous = options.get("beacon.anonymous") is not False
+
+        payload = {
+            "type": "metric",
+            "install_id": install_id,
+            "version": get_version(),
+            "docker": is_docker(),
+            "anonymous": anonymous,
+        }
+
+        if not anonymous:
+            payload["admin_email"] = options.get("system.admin-email")
```

Comment:
Not sure what kind of model you have in mind but I was thinking of just sending the `install_id` as the rest is already available from the regular pings. Alternatively, we can pool this data and then flush it altogether in a new data field in the regular pings.

What do you think?

Reply:
I was thinking we'd plumb this directly to the beacon host (getsentry) which would then send directly to our metrics and avoid keeping state in our infrastructure

Reply:
Alright, this sounds good to me.

Reply:
I added back `version`, as it made things on the back-backend cleaner.